Applications/Spire/Source/OrderImbalanceIndicator/OrderImbalanceIndicatorContextMenu.cpp

```diff
@@ -0,0 +1,226 @@
+#include "Spire/OrderImbalanceIndicator/OrderImbalanceIndicatorContextMenu.hpp"
+#include <QEvent>
+#include <QWidgetAction>
+#include "Spire/Spire/Dimensions.hpp"
+#include "Spire/Ui/CheckBox.hpp"
+#include "Spire/Ui/DropShadow.hpp"
+
+using namespace boost::signals2;
+using namespace Spire;
+
+OrderImbalanceIndicatorContextMenu::OrderImbalanceIndicatorContextMenu(
+    QWidget* parent)
+    : QMenu(parent) {
+  setWindowFlag(Qt::NoDropShadowWindowHint);
+  DropShadow context_menu_shadow(true, true, this);
+  setFixedSize(scale(140, 90));
```

Comment:
Unless you intend to parameterize these in the nearest future, move constants out to the beginning of the file so that they are easier to locate and modify if necessary.
There is no need to obsessive about that, but most important constants (like window size) probably deserve explicit declaration.

Reply:
Per Spire convention, I'll add a method in an anonymous namespace to return the size but because it relies on the DPI which is only known at runtime I can't mark it explicitly as constant.

Reply:
Well, the return value of `scale` cannot be constant, but those integer literals most certainly can. You can move those height, width constants out, or you can do what you said if you find it better.